packages/platform-sdk/test/integration-tests/sdk-v3/middlewares.test.ts

```diff
@@ -268,3 +268,31 @@ describe('Telemetry Middleware', () => {
     expect(telemetryMiddlewareCalled).toHaveBeenCalled()
   })
 })
+
+describe('Custom middleware', () => {
```

Comment:
Why do we have this test here, is it part of the work abort controller feature?

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/732